watchman/checks.py

```diff
@@ -81,3 +95,7 @@ def email():
 
 def storage():
     return {"storage": _check_storage()}
+
+
+def amqp():
+    return {'amqp': _check_ampq(settings.AMQP_CONNECTIONS)}
```

Comment:
Is this a "standard" way of configuring amqp in Django, or is this specific to how we do it? That would push me towards a separate package as well..